sotodlib/tod_ops/fft_ops.py

```diff
@@ -8,13 +8,17 @@
 import numdifftools as ndt
 import numpy as np
 import pyfftw
+import logging
```

Comment:
Instead of introducing a logger just to add the warning message use python warnings package i.e.
```
import warnings
...
warnings.warn('calc_wn will be biased. noverlap argument of welch '
                          'needs to be 0 to get unbiased median white noise estimate.')
```